fastestimator/op/tensorop/model/model.py

```diff
@@ -131,8 +134,10 @@ def forward(self, data: Union[Tensor, List[Tensor]], state: Dict[str, Any]) -> U
         return data
 
 
-def _capture_call_tf(input: tf.Tensor, fe_storage: Dict[Union[str, torch.device], Tensor],
-                     fe_layer: tf.keras.layers.Layer, **kwargs) -> tf.Tensor:
+def _capture_call_tf(input: tf.Tensor,
+                     fe_storage: Dict[Union[str, torch.device], Tensor],
```

Comment:
actually I think here we're forced to use the name 'input' in order to match the tf and torch function signatures